File: Lista_2/Exercicios_CASA-lista-2/EX6-Mes-c/EX6-c.py
num = int(input())
omes = "numero invalido"
meses = ("janeiro", "fevereiro", "março", "abril", "maio", "junho", "julho", "agosto", "setembro", "outubro", "novembro", "dezembro")
if 1 <= num <= len(meses):
    omes = meses[num - 1]
print(omes)

File: _IFSP/LP1/Lista_2/Exercicios_CASA-lista-2/EX6-Mes-c/EX6-c.c
// Online C compiler to run C program online
#include <stdio.h>

int main() {
    
    int num;
    scanf("%d", &num);
    char *meses[12] = {"Janeiro", "Fevereiro", "Março", "Abril", "Maio", "Junho", "Julho", "Agosto", "Setembro", "Outubro", "Novembro", "Dezembro"
        };
    if (1 <= num && num <= 12){
        printf("%s\n", meses[num-1]);
    }
    else{
        printf("Numero Invalido\n");
    }
    
    return 0;
}